chain-impl-mockchain/src/rewards.rs

```diff
@@ -81,6 +84,64 @@ impl TaxType {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+pub struct PoolLimit {
+    pub npools: NonZeroU32,
+    pub npools_threshold: NonZeroU32,
+}
+
+impl PoolLimit {
+    pub fn zero() -> Self {
+        PoolLimit {
+            npools: NonZeroU32::new(1).unwrap(),
+            npools_threshold: NonZeroU32::new(1).unwrap(),
+        }
+    }
+
+    pub fn serialize_in(&self, bb: ByteBuilder<Self>) -> ByteBuilder<Self> {
+        bb.u32(self.npools.get()).u32(self.npools_threshold.get())
+    }
+
+    pub fn read_frombuf(rb: &mut ReadBuf) -> Result<Self, ReadError> {
+        let npools = rb.get_u32()?;
+        let npools_threshold = rb.get_u32()?;
+
+        Ok(PoolLimit {
+            npools: NonZeroU32::new(npools).unwrap(),
+            npools_threshold: NonZeroU32::new(npools_threshold).unwrap(),
+        })
+    }
```

Comment:
This not particularly important, but just as FYI, this could be implemented by the Readable class also

Reply:
OK.  Since we're close to release, we should probably not do this at this point